common/kommander-operator/helmrelease.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: kommander-operator
+  namespace: ${releaseNamespace}
+spec:
+  dependsOn:
+    - namespace: kommander
```

Comment:
I thought we agreed not to use this dependency?

Reply:
Yeah, let's remove it

Reply:
noted, [c2b7914](https://github.com/mesosphere/kommander-applications/pull/937/commits/c2b7914c0aa79df89f5082e1460db576450881fe)